CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# Contributing
+
+Thank you for investing your time in contributing to our project! Any contribution you make will be reflected on [chunky-dev.github.io/docs/](https://chunky-dev.github.io/docs/).
+
+Read our [Code of Conduct](CODE_OF_CONDUCT.md) to keep our community approachable and respectable.
+
+In this guide you will get an overview of the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR.
+
+When contributing to this repository, please consider discussing significant changes you wish to make via an issue or the #docs channel on our [Discord server][chunky-discord] prior to making a change. Correcting spelling or grammar mistakes, or refining existing docs are not considered significant. A full rewrite of larger articles, adding new articles, new mkdoc plugins, etc. Can be considered more significant changes.
```

Comment:
Perhaps link to the docs channel? (Or create a permeant invite to the docs channel).
```suggestion
When contributing to this repository, please consider discussing significant changes you wish to make via an issue or the [#docs](https://discord.com/channels/541221265512464394/843970702637662228) channel on our [Discord server][chunky-discord] prior to making a change. Correcting spelling or grammar mistakes, or refining existing docs are not considered significant. A full rewrite of larger articles, adding new articles, new mkdoc plugins, etc. Can be considered more significant changes.
```